pages/product.js

```diff
@@ -1,53 +1,58 @@
+    async checkColorExists() {
+        return elCheck.checkElementExists(this.colorDropDown);
     },
 
-    async checkSizeExists() {
-        return await tryTo(() => I.seeElement(this.sizeDropDown));
+    async selectColor() {
+        if (await this.checkColorExists() == true) {
+            I.click(this.colorDropDown);
+            I.click(this.colorOption);
+        }
     },
 
-    async checkColorExists() {
-        return await tryTo(() => I.seeElement(this.colorDropDown));
+    async selectSize() {
+        if (await this.checkSizeExists() == true) {
+            I.click(this.sizeDropDown);
+            I.click(this.sizeOption);
+        }
     },
 
+
     async getProductPrice() {
-        if (await this.checkColorExists() == true && await this.checkSizeExists() == true) {
-            console.log('product price: ', this.parsePrice(await I.grabTextFrom(this.productPriceText)));
-            console.log('color price: ', this.parsePrice(await I.grabTextFrom(this.colorOption)));
-            console.log('size price: ', this.parsePrice(await I.grabTextFrom(this.sizeOption)));
-            return this.parsePrice(await I.grabTextFrom(this.productPriceText)) + this.parsePrice(await I.grabTextFrom(this.colorOption)) + this.parsePrice(await I.grabTextFrom(this.sizeOption));
-        }
-        else if (await this.checkColorExists() == true && await this.checkSizeExists() == false) {
-            console.log('product price: ', this.parsePrice(await I.grabTextFrom(this.productPriceText)));
-            console.log('color price: ', this.parsePrice(await I.grabTextFrom(this.colorOption)));
-            return this.parsePrice(await I.grabTextFrom(this.productPriceText)) + this.parsePrice(await I.grabTextFrom(this.colorOption));
-        } else if (await this.checkColorExists() == false && await this.checkSizeExists() == true) {
-            console.log('product price: ', this.parsePrice(await I.grabTextFrom(this.productPriceText)));
-            console.log('size price: ', this.parsePrice(await I.grabTextFrom(this.sizeOption)));
-            return this.parsePrice(await I.grabTextFrom(this.productPriceText)) + this.parsePrice(await I.grabTextFrom(this.sizeOption));
-        }else if (await this.checkColorExists() == false && await this.checkSizeExists() == false){
-            console.log('product price: ', this.parsePrice(await I.grabTextFrom(this.productPriceText)));
-            return this.parsePrice(await I.grabTextFrom(this.productPriceText));
+        let price = priceHelper.parsePrice(await I.grabTextFrom(this.productPriceText));
+        console.log('product price: ', priceHelper.parsePrice(await I.grabTextFrom(this.productPriceText)));
+        if (await this.checkColorExists() == true) {
+            console.log('color price: ', priceHelper.parsePrice(await I.grabTextFrom(this.colorOption)));
+            price += priceHelper.parsePrice(await I.grabTextFrom(this.colorOption));
+            if (await this.checkSizeExists() == true) {
```

Comment:
move size check out of color check, maybe there is a product without color, but with size